code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -136,6 +136,9 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	//Copies variables from sample
 	//Returns: 1 if we are mutable, 0 otherwise
 
+/datum/gas_mixture/proc/ratio_copy_from(datum/gas_mixture/sample, ratio = 1)

```

Comment:
can you remove all of these, dunc did this but it actually adds compiler and runtime overhead

Reply:
actually it's been done this way forever. originally when profiled it was found that this reduced runtime overhead but that turned out to be a byond bug. i was under the impression it made no runtime difference however, not increased overhead

Reply:
to clarify i know that it increases compiler overhead, i'm asking about runtime specifically

Reply:
Im scrubbing this file from this PR but if I remember I'll come back and scrub the dupe procs and move the comments down